content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: blog
+title: "My first-ever KubeCon - KubeCon+CloudNativeCon North America 2023"
+date: 2023-12-05
+slug: My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023
+---
+
+**Author:** Rajdeep Pal (Student, Techno India University)
+
+_This post is a guide for beginners to look at huge and overwhelming events like KubeCon._
+
+# Introduction
```

Comment:
```suggestion
```

We never use first level headings inside Markdown (because the theme adds one for us based on the `title`)